src/OrbitGl/App.cpp

```diff
@@ -2481,10 +2482,47 @@ Future<std::vector<ErrorMessageOr<CanceledOr<void>>>> OrbitApp::LoadAllSymbols()
 
     loading_futures.push_back(RetrieveModuleAndLoadSymbols(module));
   }
+  if (absl::GetFlag(FLAGS_auto_frame_track)) {
+    // Orbit will try to add the default frame track while loading all symbols.
+    std::ignore = AddDefaultFrameTrack();
+  }
 
   return orbit_base::WhenAll(absl::MakeConstSpan(loading_futures));
 }
 
+Future<ErrorMessageOr<void>> OrbitApp::AddDefaultFrameTrack() {
+  const std::filesystem::path default_auto_preset_folder_path =
+      orbit_base::GetExecutableDir() / "autopresets";
+  const std::filesystem::path stadia_default_preset_path =
+      default_auto_preset_folder_path / "stadia-default-frame-track.opr";
+
+  std::vector<std::filesystem::path> auto_preset_paths = {stadia_default_preset_path};
+
+  // Each preset in auto_preset_paths contains a FrameTrack that users might be interested in
+  // loading by default. Orbit will try to load automatically the first loadable preset from the
```

Comment:
Maybe worth adding a comment why we want the first loadable preset only. An alternative approach could to be load all loadable presets. But I think we need to be able to have fallbacks to other presets and there might be cases where multiple frame tracks would be loaded in presence of fallbacks (e.g. the one added here and a different Vulkan module), which is not what we want.

Reply:
Agree, thanks for the comment. As you mentioned, the problem of the alternative approach is having several FrameTrack by default, which is not desired. I'm always thinking on beginner users with this feature, and the point is having an only and potentially useful FrameTrack by default to educate them that they are there. Also in the future we are planning to use that FrameTrack to create some idea of "Frames" inside Orbit. Having 2 FrameTrack is confusing and is something that could be needed only for advanced/expert users. Potentially, we are able to extend a preset to load multiple functions/frame tracks. 

I will add that comment.